tests/platform_code/test_platform_code.py

```diff
@@ -76,20 +76,31 @@ def test_notify_quotes(
 
 
 @pytest.mark.parametrize(
-    "platform, expected_return_code",
+    "platform, return_code, expected_return_value",
     [
-        ("Linux", 0),
-        ("MacOS", 0),
-        ("WSL", 1),
+        ("Linux", 0, "success"),
+        ("MacOS", 0, "success"),
+        ("WSL", 1, "success"),
+        ("Linux", 1, "failure"),
+        ("MacOS", 1, "failure"),
+        # NOTE: WSL always returns a non-zero exit code (1), so we do not test for it.
```

Comment:
As per another comment, we should test for this, even if it results in the same response - we're testing the function, not the system behavior, in this test.

Reply:
I removed the failure handling entirely since it's considered an abnormality that might be addressed in the future. `explorer.exe` seems to be intended to return 1 for failure and 0 for success, so returning 0 for a failure didn't make sense. 

This could be updated if WSL fixes this, what do you think?